extra_requirements.txt

```diff
@@ -3,7 +3,6 @@ accelerate>=1.0
 cdsapi>=0.7.4
 earthengine-api>=0.1
 einops>=0.8
-gcsfs>=2024.10
```

Comment:
It looks like s3fs is also covered in the `fsspec[s3,gcs]` so should that be removed from here as well?